addons/models/1000-Hadoop/1100-spark_model.json

```diff
@@ -305,6 +305,34 @@
         }
       ]
     },
+    {
+      "name": "spark_application",
+      "superTypes": [
+        "Process"
+      ],
+      "serviceType": "spark",
+      "typeVersion": "1.0",
+      "attributeDefs": [
+        {
+          "name": "currUser",
```

Comment:
consider renaming "currUser" to "currentUser"

Reply:
Thanks! Changed to  "currentUser".

Reply:
Thanks! Changed to "currentUser".